packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { ToggleSwitch } from '../toggle-switch/toggle-switch';
+import { LightningIcon } from '@notional-finance/icons';
+import { Box, styled } from '@mui/material';
+import { H4 } from '../typography/typography';
+
+interface LeverUpToggleProps {
+  leveredUp: boolean;
+  handleLeverUpToggle: () => void;
+}
+
+export const LeverUpToggle = ({
+  leveredUp,
+  handleLeverUpToggle,
+}: LeverUpToggleProps) => {
+  const [isChecked, setIsChecked] = useState(leveredUp);
+
+  const handleToggle = () => {
+    handleLeverUpToggle();
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <Container onClick={() => handleToggle()}>
+      <LightningIcon viewBox="0 0 12 16" sx={{ height: '16px' }} />
+      <H4 sx={{ textWrap: 'nowrap' }}>
+        <FormattedMessage defaultMessage={'Lever Up'} />
+      </H4>
+      <ToggleSwitch
+        isChecked={isChecked}
+        onToggle={() => handleToggle()}
+        sx={{ marginRight: '-10px' }}
+      />
+    </Container>
+  );
+};
+
+const Container = styled(Box)(
+  ({ theme }) => `
+    background: linear-gradient(180deg, rgba(43, 202, 208, .4) 0%, rgba(139, 193, 229, .4) 100%);
+    padding: ${theme.spacing(1)};
+    z-index: 1;
+    border-radius: 19px;
```

Comment:
We can't use a theme border radius here?

Reply:
This is based off of the mock. borderRadiusLarge from the theme is 10px which makes the button look a lot different.